pybv/tests/test_bv_writer.py

```diff
@@ -6,18 +6,15 @@
 #          Stefan Appelhoff <stefan.appelhoff@mailbox.org>
 #
 # License: BSD (3-clause)
-
+from datetime import datetime, timezone
 import os
-import os.path as op
 from shutil import rmtree
 from tempfile import mkdtemp
-from datetime import datetime, timezone
-
-import pytest
 
 import mne
 import numpy as np
 from numpy.testing import assert_allclose, assert_array_equal
+import pytest
```

Comment:
ordering imports here, maybe we should add isort to the CI

Reply:
I did add isort and added it to the CI --> doing that early for a project leads to much more readable import sections in the long run.